File: sBuilding/Trunk/Server/Java/AccessControlService/src/com/swt/object/DoorOut.java
package com.swt.object;

import java.io.Serializable;




public class DoorOut implements Serializable  {
	
	private static final long serialVersionUID = -4438911533642453314L;
	
	private long Id;

	private long SubOrgId;

	private long MemberId;

	private long CardId;

	private long DeviceDoorId;

	private String SerialNumber;

	private String DateIn;

	private String DateOut;

	private String ImageIn;

	private String ImageOut;

	private String CreateBy;

	private String CreateAt;

	private String ModifiedBy;

	private String ModifiedAt;

	private int Status;

	public long getId() {
		return Id;
	}

	public void setId(long id) {
		Id = id;
	}

	public long getSubOrgId() {
		return SubOrgId;
	}

	public void setSubOrgId(long subOrgId) {
		SubOrgId = subOrgId;
	}

	public long getMemberId() {
		return MemberId;
	}

	public void setMemberId(long memberId) {
		MemberId = memberId;
	}

	public long getCardId() {
		return CardId;
	}

	public void setCardId(long cardId) {
		CardId = cardId;
	}

	public long getDeviceDoorId() {
		return DeviceDoorId;
	}

	public void setDeviceDoorId(long deviceDoorId) {
		DeviceDoorId = deviceDoorId;
	}

	public String getSerialNumber() {
		return SerialNumber;
	}

	public void setSerialNumber(String serialNumber) {
		SerialNumber = serialNumber;
	}

	public String getDateIn() {
		return DateIn;
	}

	public void setDateIn(String dateIn) {
		DateIn = dateIn;
	}

	public String getDateOut() {
		return DateOut;
	}

	public void setDateOut(String dateOut) {
		DateOut = dateOut;
	}

	public String getImageIn() {
		return ImageIn;
	}

	public void setImageIn(String imageIn) {
		ImageIn = imageIn;
	}

	public String getImageOut() {
		return ImageOut;
	}

	public void setImageOut(String imageOut) {
		ImageOut = imageOut;
	}

	public String getCreateBy() {
		return CreateBy;
	}

	public void setCreateBy(String createBy) {
		CreateBy = createBy;
	}

	public String getCreateAt() {
		return CreateAt;
	}

	public void setCreateAt(String createAt) {
		CreateAt = createAt;
	}

	public String getModifiedBy() {
		return ModifiedBy;
	}

	public void setModifiedBy(String modifiedBy) {
		ModifiedBy = modifiedBy;
	}

	public String getModifiedAt() {
		return ModifiedAt;
	}

	public void setModifiedAt(String modifiedAt) {
		ModifiedAt = modifiedAt;
	}

	public int getStatus() {
		return Status;
	}

	public void setStatus(int status) {
		Status = status;
	}

	public static long getSerialversionuid() {
		return serialVersionUID;
	}

}

File: sWorld/Trunk/SWTClient/CSharp/GPCoreCommon/sWorldModel/TransportData/DoorOut.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Text;

namespace sWorldModel.TransportData
{
    [DataContract]
    public class DoorOut
    {
        [DataMember]
        public long Id { get; set; }

        [DataMember]
        public long SubOrgId{ get; set; }

        [DataMember]
        public long MemberId{ get; set; }

        [DataMember]
        public long CardId{ get; set; }

        [DataMember]
        public long DeviceDoorId{ get; set; }

        [DataMember]
        public String SerialNumber{ get; set; }

        [DataMember]
        public String DateIn{ get; set; }

        [DataMember]
        public String DateOut{ get; set; }

        [DataMember]
        public String ImageIn{ get; set; }

        [DataMember]
        public String ImageOut{ get; set; }

        [DataMember]
        public String CreateBy{ get; set; }

        [DataMember]
        public String CreateAt{ get; set; }

        [DataMember]
        public String ModifiedBy{ get; set; }

        [DataMember]
        public String ModifiedAt{ get; set; }

        [DataMember]
        public int Status{ get; set; }

    }
}
